proto/stream_plan.proto

```diff
@@ -689,7 +689,7 @@ message StreamFragmentGraph {
   // edges between fragments.
   repeated StreamFragmentEdge edges = 2;
 
-  repeated uint32 dependent_relation_ids = 3;
+  repeated uint32 dependent_table_ids = 3;
```

Comment:
Does the renaming indicate that the IDs are for tables/mviews or that the structs are `TableId`s?

Reply:
Yes, the ids here are all streaming job ids. Previously the field is also used for dependency resolving and source ids were included as a workaround.